server/mailer/RegistrationMailer.js

```diff
@@ -0,0 +1,23 @@
+import { renderFile } from 'ejs'
+import path from 'path'
+import BaseMailer from './BaseMailer'
+
+export default class RegistrationMailer extends BaseMailer {
+  constructor(user) {
+    super({ to: [process.env.ADMIN_EMAIL], from: process.env.EMAIL_SENDER })
+    this.subject = 'New Registered User'
+    this.user = user
+  }
+
+  get body() {
+    const registrationEmailTemplate = path.join(
+      __dirname,
+      'server',
+      'mailer',
+      'templates',
+      'registrationTemplate.ejs'
+    )
+
+    return renderFile(registrationEmailTemplate, user)
```

Comment:
do you need to pass the user in an object to access it in the template as "user"?

```suggestion
    return renderFile(registrationEmailTemplate, { user })
```

Reply:
I don't think we need to wrap it in brackets, the user is already an object.

Reply:
but how does the template know the variable name?

Reply:
I've updated the template.
[bc9cbcf](https://github.com/AMP-SCZ/dpdash/pull/457/commits/bc9cbcfb5650ebceec9074f06d4e04dcc47fc58f)